src/test/java/redux/api/StoreTest.java

```diff
@@ -0,0 +1,82 @@
+package redux.api;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static redux.api.helpers.ActionsCreator.addTodo;
+import static redux.api.helpers.ActionsCreator.unknownAction;
+
+import org.junit.Test;
+import redux.api.helpers.Reducers;
+import redux.api.helpers.State;
+import redux.api.helpers.Todo;
+
+public abstract class StoreTest {
+
+    abstract <S> Store<S> createStore(Reducer<State> reducer, State state);
+
+    @Test
+    public void passes_the_initial_action_and_the_initial_state() {
```

Comment:
I find it easier to read like that. 

But I will not fight for that, as I agree convention is to use camelcase.